### ISSUE TITLE
Audio playback noise/static 

### ISSUE THREAD

**Issue by firstone** [FalconChristmas/fpp#2060]
# Please answer the following:
**FPP Version and Hardware**: 
8.x-master-727-g493cc437 on Debian GNU/Linux 12 (bookworm) (ProxMox VM)

**Describe the bug and Steps to reproduce**: 
A clear and concise description of what the bug is and the steps to be taken to reproduce. 
1. Uploaded several MP3 (from Amazon).
2. Status Page -> Audio -> Play
3. Every song produces highly noticeable cracking noises. Selecting something under Audio Output Format fixes it. It can be literally any option other than default. Same behavior with ugreen USB adapter and Sound Blaster 3 USB.

**Comment by patdelaney** [comment]
I don't believe we have every qualified FPP on Proxmox VM especially with for audio> I did a quick google search and came up with this that you can try.  https://forum.proxmox.com/threads/usb-audio-device-passthrough.117024/
But as far as I know what you are doing is unsupported by out team at this point.

**Comment by firstone** [comment]
The fact that it only doesn't work with "Default" but works fine with any other format, makes me think it's not proxmox related - if it was, sound wouldn't work under any circumstances. 

Audio is really sole reason to run this set up as my receiver is located in an equipment rack and it's not something I can move to outside where the controllers are. 

Since changing output format works for me as a work around, I guess I'm fine. But if there's anything I can do to help to troubleshoot, let me know. I've seen some people asking about VM on reddit, so I know I'm not the only one.

**Comment by patdelaney** [comment]
I guess I missed the Select something under audio format. Can you re-explain what you mean and add a screenshot.  Did you setup passthrough in Proxmox.  I have 3 proxmox "test" systems and can try and reproduce maybe it's just an initialization thing

**Comment by patdelaney** [comment]
They are all hosted on Dell Mini PCs

**Comment by firstone** [comment]
I have pass-through set up for 1 USB port:

<img width="405" alt="Screenshot 2024-11-21 at 8 35 16 PM" src="https://github.com/user-attachments/assets/bf752aa6-d24c-4c99-a180-fced1561d364">

It seems to work fine with something other than "Default". I've tried 44100 (mp3's are in 44.1k). And it works with 96000.

<img width="434" alt="Screenshot 2024-11-21 at 8 34 27 PM" src="https://github.com/user-attachments/assets/05072eac-e570-4fac-a2cd-e38be0e69a52">

I'm super happy there is a workaround because, otherwise, this worked out super well - installed from the .sh script with no errors on vanilla barebones Debian install, started up, connects to Kulp, connects to ESPixelStick, everything is great.
